_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,135 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
```

Comment:
The "content model" is something from HTML, not from ARIA. So it only cares about HTML element. Technically no matter what their role, but we might want to keep the role clause in order to flag accessibility issues.

The corresponding thing for ARIA is the "required context role (or owned elements)" for which we also have a rule.

Reply:
Thanks for this @Jym77 

I get that. And I guess it could be argued then that the ARIA example above would be tested using the other rules. For example [ARIA required context role](https://act-rules.github.io/rules/ff89c9), passed examples 1, 2, and 3.

I think we can do more to illustrate this difference, as @carlosapaduarte was suggesting in other comment. Maybe adding "HTML content model" to the existing phrasing, and maybe adding ARIA examples in the inapplicable section to demonstrate that in this rule we are only focusing in HTML native elements.